### PR TITLE
General: Small code cleanups

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Changelog Description
Small code cleanup and updates.

## Additional info
Removed unused import in ftrack login. Refactored code in muster. Removed `default` keyword argument from `get_publish_instance_label`. Overlay message always updates the style, to be able unset it. Updated mockup environment variables needed for custom scripts.

## Testing notes:
Go through changes and validate them. Everything should work as before.

### CODE REVIEW

File: openpype/modules/muster/muster.py [hunk 1]
```diff
@@ -110,16 +112,10 @@ def get_auth_token(self, username, password):
...
-        data = {
-            'token': token
-        }
+        """Save credentials to JSON file."""
 
-        file = open(self.cred_path, 'w')
-        file.write(json.dumps(data))
-        file.close()
+        with open(self.cred_path, 'w') as stream:
+            stream.write(json.dumps({'token': token}))
```
[the remark this answers is not in the document]
Comment (iLLiCiTiT) [reply openpype/modules/muster/muster.py:118]: True :)

File: openpype/modules/muster/muster.py [hunk 2]
```diff
@@ -110,16 +112,10 @@ def get_auth_token(self, username, password):
...
-        """
-        data = {
-            'token': token
-        }
+        """Save credentials to JSON file."""
 
-        file = open(self.cred_path, 'w')
-        file.write(json.dumps(data))
-        file.close()
+        with open(self.cred_path, "w") as f:
```
Comment (BigRoy) [inline openpype/modules/muster/muster.py:117]:
```suggestion
        with open(self.cred_path, 'w') as f:
```

I swapped the quote style to double quotes in my suggestion but it seems that this python file uses single quotes the whole time so it felt a bit mismatched. :) Maybe preserve this with single quotes then too.

**Review by kalisp — APPROVED** [review]
LGTM

**Review by BigRoy — APPROVED** [review]
I didn't do a test run, but the code changes seem fine.

**Pull request merged**